master/internal/db/postgres_checkpoints.go

```diff
@@ -149,20 +149,20 @@ func UpdateCheckpointSizeTx(ctx context.Context, idb bun.IDB, checkpoints []uuid
 	err := idb.NewRaw(`
 UPDATE runs SET checkpoint_size=sub.size, checkpoint_count=sub.count FROM (
 	SELECT
-		trial_id,
+		run_id,
 		COALESCE(SUM(size) FILTER (WHERE state != 'DELETED'), 0) AS size,
 		COUNT(*) FILTER (WHERE state != 'DELETED') AS count
 	FROM checkpoints_v2
-	JOIN trial_id_task_id tt ON tt.task_id = checkpoints_v2.task_id
+	JOIN run_id_task_id tt ON tt.task_id = checkpoints_v2.task_id
```

Comment:
shouldn't this join on runs not run_id_task_id?

Reply:
yes it should be, updated this